diff --git a/base/BUILD.gn b/base/BUILD.gn
index 462b75bfb8a3ed5f389282dc6c83262baf953a83..c0c094e23bd28c82f52259feea0937afa68e8a5d 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4396,6 +4396,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
       "android/java/src/org/chromium/base/WrappedClassLoader.java",
     ]
+    sources += brave_java_base_sources
 
     if (use_clang_profiling) {
       sources += [
@@ -4860,6 +4861,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("base_cached_flags_java") {
@@ -4885,6 +4887,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/cached_flags/ValuesOverridden.java",
       "android/java/src/org/chromium/base/cached_flags/ValuesReturned.java",
     ]
+    sources += brave_base_cached_flags_java_sources
   }
 
   android_aidl("process_launcher_aidl") {
