diff --git a/base/test/BUILD.gn b/base/test/BUILD.gn
index 1a1713af3af0ef723afba320c4a70a1bb17523f5..879d9729e1ae0ca46fad8a4cd172081ef313eea9 100644
--- a/base/test/BUILD.gn
+++ b/base/test/BUILD.gn
@@ -346,6 +346,7 @@ static_library("test_support") {
       "trace_to_file.h",
     ]
   }
+  import("//brave/base/sources.gni") sources += brave_base_test_support_sources
 }
 
 component("proto_test_support") {
