diff --git a/base/threading/thread_restrictions.h b/base/threading/thread_restrictions.h
index 965206e18e7a536947354c734ef17196c3e736ff..b80a653a79062a94281dcaf5d1d136de2432cfe4 100644
--- a/base/threading/thread_restrictions.h
+++ b/base/threading/thread_restrictions.h
@@ -733,6 +733,7 @@ class BASE_EXPORT ScopedAllowBaseSyncPrimitives {
   ScopedAllowBaseSyncPrimitives& operator=(
       const ScopedAllowBaseSyncPrimitives&) = delete;
 
+  BRAVE_SCOPED_ALLOW_BASE_SYNC_PRIMITIVES_H
  private:
   // This can only be instantiated by friends. Use
   // ScopedAllowBaseSyncPrimitivesForTesting in unit tests to avoid the friend
