diff --git a/base/BUILD.gn b/base/BUILD.gn
index 0eb84c31ebbda5e7e842cd0f0df5ff2a15048178..1309a343bdee04276a38919ce14edcefb41f76c9 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4466,6 +4466,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
       "android/java/src/org/chromium/base/WrappedClassLoader.java",
     ]
+    sources += brave_java_base_sources
 
     if (use_clang_profiling) {
       sources += [
@@ -4934,6 +4935,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   # Most cached_flags code is in components/cached_flags. These classes are
