diff --git a/base/test/BUILD.gn b/base/test/BUILD.gn
index 8c97b5b9dfebd4a8ecc2729313cc6b256aff2d30..b15dfbe8e5c087edb979646e5e4f7240764b6264 100644
--- a/base/test/BUILD.gn
+++ b/base/test/BUILD.gn
@@ -363,6 +363,7 @@ static_library("test_support") {
       "trace_to_file.h",
     ]
   }
+  import("//brave/base/sources.gni") sources += brave_base_test_support_sources
 }
 
 component("proto_test_support") {
