diff --git a/base/logging.cc b/base/logging.cc
index 508ce135131aadcc6cc69a401b63124da520e895..16768efed55f78ef8508bdabeeab27da97a7acab 100644
--- a/base/logging.cc
+++ b/base/logging.cc
@@ -525,10 +525,9 @@ bool BaseInitLoggingImpl(const LoggingSettings& settings) {
     GetScopedFxLogger() = base::ScopedFxLogger::CreateForProcess();
   }
 #endif
-
 #if !BUILDFLAG(IS_NACL)
   // Connects Rust logging with the //base logging functionality.
-  internal::init_rust_log_crate();
+  // internal::init_rust_log_crate();
 #endif
 
   // Ignore file options unless logging to file is set.
