diff --git a/base/BUILD.gn b/base/BUILD.gn
index 8ea4d54304dae89db570e960160991da0949eeb0..c48f4012b9168a22620978acacde6614c8ea85d8 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4484,6 +4484,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
       "android/java/src/org/chromium/base/WrappedClassLoader.java",
     ]
+    sources += brave_java_base_sources
 
     if (use_clang_profiling) {
       sources += [
@@ -4947,6 +4948,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   # Most cached_flags code is in components/cached_flags. These classes are
