diff --git a/base/metrics/histogram_functions.h b/base/metrics/histogram_functions.h
index e460fe8e564fd2898bc6b73e4b0c1a02334df70b..8eab0ead876e5e89dc14dfd98ffc87f2262b18e6 100644
--- a/base/metrics/histogram_functions.h
+++ b/base/metrics/histogram_functions.h
@@ -79,6 +79,7 @@ void UmaHistogramEnumeration(std::string_view name, T sample) {
   static_assert(static_cast<uintmax_t>(T::kMaxValue) <=
                     static_cast<uintmax_t>(INT_MAX) - 1,
                 "Enumeration's kMaxValue is out of range of INT_MAX!");
+  BRAVE_HISTOGRAM_FUNCTIONS_UMA_HISTOGRAM_ENUMERATION
   DCHECK_LE(static_cast<uintmax_t>(sample),
             static_cast<uintmax_t>(T::kMaxValue));
   return UmaHistogramExactLinear(name, static_cast<int>(sample),
