diff --git a/chrome/browser/ui/browser_navigator.cc b/chrome/browser/ui/browser_navigator.cc
index d4d5759c254e51b35b8dfb0eda2540009f80572e..a3f55773b98752aec010365542754bcb60cd0da6 100644
--- a/chrome/browser/ui/browser_navigator.cc
+++ b/chrome/browser/ui/browser_navigator.cc
@@ -152,7 +152,7 @@ bool AdjustNavigateParamsForURL(NavigateParams* params) {
   // technically an incognito window, these URLs are allowed.
   Profile* profile = params->initiating_profile;
   if (!params->contents_to_insert && !params->switch_to_singleton_tab &&
-      !IsURLAllowedInIncognito(params->url) && !profile->IsGuestSession() &&
+      !IsURLAllowedInIncognito(params->url, profile) && !profile->IsGuestSession() &&
       (profile->IsOffTheRecord() ||
        params->disposition == WindowOpenDisposition::OFF_THE_RECORD)) {
     profile = profile->GetOriginalProfile();
@@ -664,6 +664,7 @@ base::WeakPtr<content::NavigationHandle> Navigate(NavigateParams* params) {
       return nullptr;
     }
   }
+  BRAVE_ADJUST_NAVIGATE_PARAMS_FOR_URL
 
   // Open System Apps in their standalone window if necessary.
   // TODO(crbug.com/40136163): Remove this code after we integrate with intent
@@ -1033,7 +1034,7 @@ base::WeakPtr<content::NavigationHandle> Navigate(NavigateParams* params) {
   return navigation_handle;
 }
 
-bool IsURLAllowedInIncognito(const GURL& url) {
+bool IsURLAllowedInIncognito(const GURL& url, content::BrowserContext* browser_context) {
   if (url.scheme() == content::kViewSourceScheme) {
     // A view-source URL is allowed in incognito mode only if the URL itself
     // is allowed in incognito mode. Remove the "view-source:" from the start
@@ -1047,8 +1048,9 @@ bool IsURLAllowedInIncognito(const GURL& url) {
     if (stripped_url.is_empty()) {
       return true;
     }
-    return stripped_url.is_valid() && IsURLAllowedInIncognito(stripped_url);
+    return stripped_url.is_valid() && IsURLAllowedInIncognito(stripped_url, browser_context);
   }
 
+  BRAVE_IS_URL_ALLOWED_IN_INCOGNITO
   return IsHostAllowedInIncognito(url);
 }
