diff --git a/chrome/browser/hub/internal/BUILD.gn b/chrome/browser/hub/internal/BUILD.gn
index e8677ef73b1a28bec168e92a62c6e279e4b27d91..3a7b18a2361c048e7c20d0af35b64a44b9854d33 100644
--- a/chrome/browser/hub/internal/BUILD.gn
+++ b/chrome/browser/hub/internal/BUILD.gn
@@ -51,6 +51,7 @@ android_library("java") {
     "android/java/src/org/chromium/chrome/browser/hub/TranslateHubLayoutAnimationFactory.java",
     "android/java/src/org/chromium/chrome/browser/hub/TranslateHubLayoutAnimationFactoryImpl.java",
   ]
+  sources += brave_browser_hub_internal_java_sources
   deps = [
     ":java_resources",
     "//base:base_java",
