diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index dd5c9db43d693fb9df7b5a0d661b4e8999956c49..184111cbf5d07eaf75a499def8c9d1d5eee37a08 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2516,6 +2516,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
