diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index f4f8e674afa03e90c5089ddbab388b17b484752b..458c5021c6b99e5a3d50000ac08b46553c70fabe 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2529,6 +2529,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
@@ -4256,7 +4257,6 @@ static_library("browser") {
       "//chrome/browser/apps/link_capturing:features",
       "//chrome/browser/companion/core",
       "//chrome/browser/companion/visual_query",
-      "//chrome/browser/enterprise/connectors/analysis:features",
       "//chrome/browser/enterprise/signals:utils",
       "//chrome/browser/feedback:feedback_enum",
       "//chrome/browser/icon_transcoder",
@@ -6745,6 +6745,7 @@ static_library("browser") {
       "enterprise/connectors/connectors_service.h",
     ]
     deps += [ "//components/enterprise/connectors/core" ]
+    deps += [ "//chrome/browser/enterprise/connectors/analysis:features" ]
   }
 
   if (enterprise_cloud_content_analysis) {
