diff --git a/chrome/browser/ui/views/profiles/profile_menu_view_base.cc b/chrome/browser/ui/views/profiles/profile_menu_view_base.cc
index ea6e921d5ee152e9c8b39973f3abfa92cbc3804e..ba6ce6d197f3abaa7ef52a694cb9a0f123183f3b 100644
--- a/chrome/browser/ui/views/profiles/profile_menu_view_base.cc
+++ b/chrome/browser/ui/views/profiles/profile_menu_view_base.cc
@@ -588,6 +588,7 @@ void ProfileMenuViewBase::BuildProfileBackgroundContainer(
     identity_info_color_callback_ = base::BindRepeating(
         &ProfileMenuViewBase::BuildIdentityInfoColorCallback,
         base::Unretained(this));
+    BRAVE_PROFILE_MENU_VIEW_BASE_SET_BACKGROUND
   } else {
     DCHECK_EQ(SK_ColorTRANSPARENT, background_color);
     profile_background_container_->SetBackground(
