diff --git a/chrome/android/java/src/org/chromium/chrome/browser/notifications/NotificationPlatformBridge.java b/chrome/android/java/src/org/chromium/chrome/browser/notifications/NotificationPlatformBridge.java
index b7f6816f7e83db69f34dab7f2b4d4a51da540cc2..2708ae35b0e4b8e535d5a9bae6c2000c6b739122 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/notifications/NotificationPlatformBridge.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/notifications/NotificationPlatformBridge.java
@@ -141,7 +141,7 @@ public class NotificationPlatformBridge {
     private TrustedWebActivityClient mTwaClient;
 
     /** Encapsulates attributes that identify a notification and where it originates from. */
-    private static class NotificationIdentifyingAttributes {
+    public static class NotificationIdentifyingAttributes {
         public final String notificationId;
         public final @NotificationType int notificationType;
         public final String origin;
@@ -896,7 +896,7 @@ public class NotificationPlatformBridge {
                 .storeNotificationResourcesIfSuspended(notification);
     }
 
-    private static NotificationBuilderBase prepareNotificationBuilder(
+    protected static NotificationBuilderBase prepareNotificationBuilder(
             NotificationIdentifyingAttributes identifyingAttributes,
             boolean vibrateEnabled,
             String title,
