diff --git a/chrome/browser/tab_ui/android/BUILD.gn b/chrome/browser/tab_ui/android/BUILD.gn
index cd8a482e0794bfd5ae868e70ba6e56a313829cbc..9fd63dbea22646299eaf73d21e7e6cf23d3a80b2 100644
--- a/chrome/browser/tab_ui/android/BUILD.gn
+++ b/chrome/browser/tab_ui/android/BUILD.gn
@@ -39,6 +39,7 @@ android_library("java") {
     "java/src/org/chromium/chrome/browser/tab_ui/TabUiThemeUtils.java",
     "java/src/org/chromium/chrome/browser/tab_ui/ThumbnailProvider.java",
   ]
+  sources += brave_tab_ui_sources
   deps = [
     ":java_resources",
     "//base:base_java",
