diff --git a/chrome/browser/ui/webui/extensions/extensions_ui.cc b/chrome/browser/ui/webui/extensions/extensions_ui.cc
index a6326a1c56aac34368fd420048f2b05ce252f6cd..04565f16c503a65f90693ebc91ff509eaccd0244 100644
--- a/chrome/browser/ui/webui/extensions/extensions_ui.cc
+++ b/chrome/browser/ui/webui/extensions/extensions_ui.cc
@@ -483,6 +483,7 @@ content::WebUIDataSource* CreateAndAddExtensionsSource(Profile* profile,
                      crosapi::browser_util::IsLacrosEnabled());
 #endif
 
+  BRAVE_CREATE_EXTENSIONS_SOURCE
   return source;
 }
 
