diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index c413874f8b098728835b6521bb824f6ed492624f..ec722db79fcbfaf96c9225d036f28e28b00f959e 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -602,6 +602,7 @@ static_library("test_support") {
     "//ui/gl",
   ]
 
+  import("//brave/chromium_src/chrome/test/sources.gni") deps += brave_chromium_src_chrome_test_test_support_deps sources += brave_chromium_src_chrome_test_test_support_sources
   data = []
   data_deps = []
 
