diff --git a/chrome/browser/ui/android/toolbar/BUILD.gn b/chrome/browser/ui/android/toolbar/BUILD.gn
index 698cbaffcb1cc55b49c6938607e16ec8edaf577f..0aeea22f3735f534f280ee350bf0e257d72253a7 100644
--- a/chrome/browser/ui/android/toolbar/BUILD.gn
+++ b/chrome/browser/ui/android/toolbar/BUILD.gn
@@ -122,6 +122,7 @@ android_library("java") {
     "java/src/org/chromium/chrome/browser/toolbar/top/tab_strip/StripVisibilityState.java",
     "java/src/org/chromium/chrome/browser/toolbar/top/tab_strip/TabStripTransitionCoordinator.java",
   ]
+  sources += brave_browser_ui_android_toolbar_sources
 
   deps = [
     ":java_resources",
