diff --git a/chrome/browser/ui/autofill/chrome_autofill_client.h b/chrome/browser/ui/autofill/chrome_autofill_client.h
index e200840106a545f4170b3c24b135e172238df142..c9c68157e352ee7cc9d32acbc47d048f801a6be2 100644
--- a/chrome/browser/ui/autofill/chrome_autofill_client.h
+++ b/chrome/browser/ui/autofill/chrome_autofill_client.h
@@ -107,9 +107,9 @@ class ChromeAutofillClient : public ContentAutofillClient,
   scoped_refptr<network::SharedURLLoaderFactory> GetURLLoaderFactory() final;
   AutofillCrowdsourcingManager& GetCrowdsourcingManager() final;
   VotesUploader& GetVotesUploader() final;
-  AutofillOptimizationGuide* GetAutofillOptimizationGuide() const final;
+  AutofillOptimizationGuide* GetAutofillOptimizationGuide() const override;
   FieldClassificationModelHandler* GetAutofillFieldClassificationModelHandler()
-      final;
+      override;
   FieldClassificationModelHandler*
   GetPasswordManagerFieldClassificationModelHandler() final;
   PersonalDataManager& GetPersonalDataManager() final;
@@ -172,10 +172,10 @@ class ChromeAutofillClient : public ContentAutofillClient,
   void TriggerUserPerceptionOfAutofillSurvey(
       FillingProduct filling_product,
       const std::map<std::string, std::string>& field_filling_stats_data) final;
-  bool IsAutofillEnabled() const final;
+  bool IsAutofillEnabled() const override;
   bool IsAutofillProfileEnabled() const final;
   bool IsAutofillPaymentMethodsEnabled() const final;
-  bool IsAutocompleteEnabled() const final;
+  bool IsAutocompleteEnabled() const override;
   bool IsPasswordManagerEnabled() const final;
   void DidFillForm(AutofillTriggerSource trigger_source, bool is_refill) final;
   bool IsContextSecure() const final;
