diff --git a/chrome/browser/resources/extensions/BUILD.gn b/chrome/browser/resources/extensions/BUILD.gn
index 1d7da12555ecf7a58fa5e7a3967062c18bf2a156..bd955d7fb554a1f719aed6d2704a7a3416e6ab9a 100644
--- a/chrome/browser/resources/extensions/BUILD.gn
+++ b/chrome/browser/resources/extensions/BUILD.gn
@@ -115,4 +115,5 @@ build_webui("build") {
     optimize_webui_host = "extensions"
     optimize_webui_in_files = [ "extensions.js" ]
   }
+  import("//brave/browser/resources/extensions/sources.gni") web_component_files += brave_extensions_web_component_files non_web_component_files += brave_extensions_non_web_component_files exclude_ts_preprocess_files = brave_extensions_local_ts_files exclude_html_css_preprocess_files = brave_extensions_local_html_files preprocess_deps = brave_extensions_preprocess_extra_deps
 }
