diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index fe7c8a4012c95545d8c0730770d5e9d3aa572f21..d5d92a61a0fcb250cf0623259d853bdd57efc3a5 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4169,6 +4169,7 @@ const FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
