diff --git a/chrome/browser/ui/omnibox/chrome_omnibox_client.h b/chrome/browser/ui/omnibox/chrome_omnibox_client.h
index c410a84a3d6e68c5c8ff70c1baaebe0fd0d79b79..57c1460f93754deb76cca40e315a66085f177ec7 100644
--- a/chrome/browser/ui/omnibox/chrome_omnibox_client.h
+++ b/chrome/browser/ui/omnibox/chrome_omnibox_client.h
@@ -23,7 +23,7 @@ class GURL;
 class LocationBar;
 class Profile;
 
-class ChromeOmniboxClient final : public OmniboxClient {
+class ChromeOmniboxClient : public OmniboxClient {
  public:
   ChromeOmniboxClient(LocationBar* location_bar,
                       Browser* browser,
