diff --git a/chrome/browser/safe_browsing/BUILD.gn b/chrome/browser/safe_browsing/BUILD.gn
index 386e4b6b6e8a9bc70408cc6a143c5f2c721fa03f..dbdfe9af4c78ef0b528b5dd68ff8d940ce5ab5b3 100644
--- a/chrome/browser/safe_browsing/BUILD.gn
+++ b/chrome/browser/safe_browsing/BUILD.gn
@@ -483,6 +483,7 @@ static_library("safe_browsing") {
         "//components/safe_browsing/android:safe_browsing_mobile",
       ]
     }
+    deps += [ "//brave/browser/ui:ui_public_dependencies" ] # safe_browsing should depend on //chrome/browser/ui but it would cause a circular dependency
   }
 }
 
