diff --git a/chrome/browser/ui/webui/BUILD.gn b/chrome/browser/ui/webui/BUILD.gn
index 8e2f7fdac0163820ad9b3a76f0765145509b38f2..869627d49a34c03855305283f2625d8e6cd51ce2 100644
--- a/chrome/browser/ui/webui/BUILD.gn
+++ b/chrome/browser/ui/webui/BUILD.gn
@@ -57,6 +57,7 @@ source_set("configs") {
   if (is_chromeos_ash) {
     deps += [ "//chrome/browser/ui/webui/ash/config" ]
   }
+  import("//brave/browser/ui/webui/sources.gni") deps += brave_browser_ui_webui_configs_deps
 }
 
 source_set("webui_util") {
