diff --git a/chrome/browser/bookmarks/android/BUILD.gn b/chrome/browser/bookmarks/android/BUILD.gn
index 80c6bcf8e2ff4fb97774f8f5c50c1a1e5c8eeb75..b957bd3931074b1e0c7b68a58d0351b22fddb8be 100644
--- a/chrome/browser/bookmarks/android/BUILD.gn
+++ b/chrome/browser/bookmarks/android/BUILD.gn
@@ -92,6 +92,7 @@ android_library("java") {
     "java/src/org/chromium/chrome/browser/bookmarks/bar/BookmarkBarViewBinder.java",
     "java/src/org/chromium/chrome/browser/bookmarks/bar/BookmarkBarVisibilityProvider.java",
   ]
+  sources += brave_browser_bookmarks_android_sources
 
   deps = [
     "//base:base_java",
