diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 27bf296e12fcc9db20a9b77ea4634bda3745e489..2e38634fb7c8a11b21229455e1928177cb330a03 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4094,6 +4094,7 @@ const FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
