diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index b53bbfec3402baf8ae7730c7c61ced05e07e9750..d3f1fe8e19f08445a7ce5d1f82dc4ea7cfc512f8 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4437,6 +4437,7 @@ const FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
