diff --git a/chrome/browser/ui/autofill/chrome_autofill_client.h b/chrome/browser/ui/autofill/chrome_autofill_client.h
index 75caee23939944e78bc1b3438b5fee065af7c825..3caad2eaea4eee6ff6c7cad0edba1b3a7c140b6c 100644
--- a/chrome/browser/ui/autofill/chrome_autofill_client.h
+++ b/chrome/browser/ui/autofill/chrome_autofill_client.h
@@ -105,9 +105,9 @@ class ChromeAutofillClient : public ContentAutofillClient,
   bool IsOffTheRecord() const final;
   scoped_refptr<network::SharedURLLoaderFactory> GetURLLoaderFactory() final;
   AutofillCrowdsourcingManager* GetCrowdsourcingManager() final;
-  AutofillOptimizationGuide* GetAutofillOptimizationGuide() const final;
+  AutofillOptimizationGuide* GetAutofillOptimizationGuide() const override;
   FieldClassificationModelHandler* GetAutofillFieldClassificationModelHandler()
-      final;
+      override;
   FieldClassificationModelHandler*
   GetPasswordManagerFieldClassificationModelHandler() final;
   PersonalDataManager& GetPersonalDataManager() final;
@@ -170,10 +170,10 @@ class ChromeAutofillClient : public ContentAutofillClient,
   void TriggerUserPerceptionOfAutofillSurvey(
       FillingProduct filling_product,
       const std::map<std::string, std::string>& field_filling_stats_data) final;
-  bool IsAutofillEnabled() const final;
+  bool IsAutofillEnabled() const override;
   bool IsAutofillProfileEnabled() const final;
   bool IsAutofillPaymentMethodsEnabled() const final;
-  bool IsAutocompleteEnabled() const final;
+  bool IsAutocompleteEnabled() const override;
   bool IsPasswordManagerEnabled() const final;
   void DidFillForm(AutofillTriggerSource trigger_source, bool is_refill) final;
   bool IsContextSecure() const final;
