diff --git a/chrome/browser/ui/webui/BUILD.gn b/chrome/browser/ui/webui/BUILD.gn
index 281c5e7bf595a0f580542b8a53dd4d08ce08c598..4cc96211c37660dfd4817d552c0be48dcc5b2178 100644
--- a/chrome/browser/ui/webui/BUILD.gn
+++ b/chrome/browser/ui/webui/BUILD.gn
@@ -61,6 +61,7 @@ source_set("configs") {
       "//chromeos/constants",
     ]
   }
+  import("//brave/browser/ui/webui/sources.gni") deps += brave_browser_ui_webui_configs_deps
 }
 
 source_set("internal_webui_config") {
