diff --git a/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc b/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc
index 0a2b31377401ef9bdfaceeb51d7cbe55c947f337..3735429dd6ce211038892e32c9974aeeb318675d 100644
--- a/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc
+++ b/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc
@@ -815,6 +815,7 @@ SkPath IconLabelBubbleView::GetHighlightPath() const {
       SkIntToScalar(radii.lower_right()), SkIntToScalar(radii.lower_right()),
       SkIntToScalar(radii.lower_left()),  SkIntToScalar(radii.lower_left())};
 
+  BRAVE_ICON_LABEL_BUBBLE_VIEW_GET_HIGHLIGHT_PATH
   return SkPath().addRoundRect(rect, sk_radii);
 }
 
