diff --git a/chrome/browser/resources/downloads/downloads.ts b/chrome/browser/resources/downloads/downloads.ts
index bb295d55818aa3c98e402c44e083e2422fa999f9..28b5b068e2c3d81d2c43f9a68f23bb2a9700315e 100644
--- a/chrome/browser/resources/downloads/downloads.ts
+++ b/chrome/browser/resources/downloads/downloads.ts
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import './brave_overrides/index.js';
 import './manager.js';
 
 export {CrToastManagerElement} from 'chrome://resources/cr_elements/cr_toast/cr_toast_manager.js';
