diff --git a/chrome/browser/profiles/profile.cc b/chrome/browser/profiles/profile.cc
index 12db3752da72dffa597b164b5e1089be0a970c1a..3c8ec911f2d4c12f9de5acc53a3371f3506d77ad 100644
--- a/chrome/browser/profiles/profile.cc
+++ b/chrome/browser/profiles/profile.cc
@@ -109,6 +109,7 @@ bool Profile::OTRProfileID::AllowsBrowserWindows() const {
   // DevTools::BrowserContext, MediaRouter::Presentation, and
   // CaptivePortal::Signin are exceptions to this ban.
   if (*this == PrimaryID() ||
+      BRAVE_ALLOWS_BROWSER_WINDOWS
       base::StartsWith(profile_id_, kDevToolsOTRProfileIDPrefix,
                        base::CompareCase::SENSITIVE) ||
       base::StartsWith(profile_id_, kMediaRouterOTRProfileIDPrefix,
