diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 4a2792eeab4283b48fc768f1bd081f840e5bb379..585a6882a25433a1825e1d3688b2cc98838020e9 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4177,6 +4177,7 @@ const FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
