diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index bc1e4d575d63b44bd1324aa36c9ed03ee53d1730..97b59fe2e2155ef3265accdff5dfedc069abdc06 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4369,6 +4369,7 @@ const FeatureEntry::FeatureVariation kMerchantTrustVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
