diff --git a/chrome/browser/ui/webui/BUILD.gn b/chrome/browser/ui/webui/BUILD.gn
index 413bcda313a3222308859d83ca89f9ef0699f44b..f2f290c34f42f850442824c415be3ce81e3d5f51 100644
--- a/chrome/browser/ui/webui/BUILD.gn
+++ b/chrome/browser/ui/webui/BUILD.gn
@@ -61,6 +61,7 @@ source_set("configs") {
       "//chromeos/constants",
     ]
   }
+  import("//brave/browser/ui/webui/sources.gni") deps += brave_browser_ui_webui_configs_deps
 }
 
 source_set("internal_webui_config") {
