diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index d7e7f17b2fa2119a3781e0bc085988284bdce814..e4b27dc8547d4f19de05dbe8f944ddf3e948e5e6 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -579,6 +579,7 @@ static_library("test_support") {
     "//ui/gl",
   ]
 
+  import("//brave/chromium_src/chrome/test/sources.gni") deps += brave_chromium_src_chrome_test_test_support_deps sources += brave_chromium_src_chrome_test_test_support_sources
   data = []
   data_deps = []
 
