diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index 1fe0ea422d36b3ba803207cd0a1234dce4c5c3c2..ecca8251e79affa011ae2598f494c0b9ff3a9633 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -573,6 +573,7 @@ static_library("test_support") {
     "//ui/gl",
   ]
 
+  import("//brave/chromium_src/chrome/test/sources.gni") deps += brave_chromium_src_chrome_test_test_support_deps sources += brave_chromium_src_chrome_test_test_support_sources
   data = []
   data_deps = []
 
@@ -3300,7 +3301,6 @@ if (!is_android) {
           "../browser/safe_browsing/cloud_content_scanning/cloud_binary_upload_service_browsertest.cc",
           "../browser/safe_browsing/download_protection/deep_scanning_browsertest.cc",
         ]
-        deps += [ "../browser/enterprise/connectors/analysis:features" ]
       }
     }
 
@@ -9611,6 +9611,7 @@ test("unit_tests") {
       "../browser/enterprise/connectors/connectors_service_unittest.cc",
     ]
     deps += [ "//components/enterprise/connectors/core" ]
+    deps += [ "//chrome/browser/enterprise/connectors/analysis:features" ]
   }
 
   if (enterprise_cloud_content_analysis) {
