diff --git a/chrome/browser/resources/settings/site_settings/constants.ts b/chrome/browser/resources/settings/site_settings/constants.ts
index 8436627f132985651fa640e48803e56808399ec3..959cf6904aca50fcf155b4c093fdd3f0a504bc99 100644
--- a/chrome/browser/resources/settings/site_settings/constants.ts
+++ b/chrome/browser/resources/settings/site_settings/constants.ts
@@ -67,6 +67,7 @@ export enum ContentSettingsTypes {
   PDF_DOCUMENTS = 'pdfDocuments',
   SITE_DATA = 'site-data',
   OFFER_WRITING_HELP = 'offer-writing-help',
+  AUTOPLAY = 'autoplay', ETHEREUM = 'ethereum', SOLANA = 'solana', BRAVE_SHIELDS = 'braveShields', GOOGLE_SIGN_IN = 'googleSignIn', LOCALHOST_ACCESS = 'localhostAccess',
 }
 
 /**
