diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index 32049c3f17b1c9a73a68e31fb19f574cdf6f5d19..015ba02c043e2b9318db6f74d35c0632d46c094b 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -1475,6 +1475,7 @@ source_set("extensions") {
       "//components/enterprise/data_controls/core/browser",
     ]
   }
+  deps += brave_extensions_resources
 }
 
 # KeyedService factories initialization.
