diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 3d5316a23270e5435ed674156064319a79beb575..4e6a494fb37d2604339c62604657b7c02f3c722e 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4159,6 +4159,7 @@ const FeatureEntry::FeatureVariation kClayBlockingVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
