diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 5f4087208f26afe12dea9b13a1e77e5ece72b67e..33f5bff4a130d091ac30691767e5a701bc79b1ed 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4472,6 +4472,7 @@ const FeatureEntry::FeatureVariation kAutofillShowTypePredictionsVariations[] =
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
