diff --git a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
index 705c8c7eafb615fdb1b9ccc9bb10986f729c9386..5d1c729c88f025c51f464856e805cfd70dae68fb 100644
--- a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
+++ b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
@@ -143,6 +143,7 @@ class AvatarToolbarButton : public ToolbarButton {
   CreateScopedZeroDelayOverrideSigninPendingTextForTesting();
 
  private:
+  BRAVE_AVATAR_BUTTON_H
   FRIEND_TEST_ALL_PREFIXES(AvatarToolbarButtonTest,
                            HighlightMeetsMinimumContrast);
 
