diff --git a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
index 2a9c4b2ef984ed27b066ed670041a31d14084220..284691c058cbc18579ba249d2d2caa4ea9bfa6b6 100644
--- a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
+++ b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
@@ -151,6 +151,7 @@ class AvatarToolbarButton : public ToolbarButton {
   CreateScopedZeroDelayOverrideSigninPendingTextForTesting();
 
  private:
+  BRAVE_AVATAR_BUTTON_H
   FRIEND_TEST_ALL_PREFIXES(AvatarToolbarButtonTest,
                            HighlightMeetsMinimumContrast);
 
