diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index f1929da7b0c54281bb3287153e5d3442d41b55b8..2daf198fa746cda2cd3ca2dfd31d69a5ffb6a788 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2516,6 +2516,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
@@ -4248,7 +4249,6 @@ static_library("browser") {
       "//chrome/browser/apps/link_capturing:features",
       "//chrome/browser/companion/core",
       "//chrome/browser/companion/visual_query",
-      "//chrome/browser/enterprise/connectors/analysis:features",
       "//chrome/browser/enterprise/signals:utils",
       "//chrome/browser/feedback:feedback_enum",
       "//chrome/browser/icon_transcoder",
@@ -6716,6 +6716,7 @@ static_library("browser") {
       "enterprise/connectors/connectors_service.h",
     ]
     deps += [ "//components/enterprise/connectors/core" ]
+    deps += [ "//chrome/browser/enterprise/connectors/analysis:features" ]
   }
 
   if (enterprise_cloud_content_analysis) {
