diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index d838fe5a7a1ae4a89cb0d1a6004a104e15ff21c3..52a53ba34184e6a528fbb157738ca141606935f7 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -152,6 +152,7 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
 #endif
     case RequestType::kIdentityProvider:
       return RequestTypeForUma::PERMISSION_IDENTITY_PROVIDER;
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
@@ -177,6 +178,7 @@ RequestTypeForUma GetUmaValueForRequests(
     return RequestTypeForUma::MULTIPLE_KEYBOARD_AND_POINTER_LOCK;
   }
 #endif
+  BRAVE_GET_UMA_VALUE_FOR_REQUESTS
   return RequestTypeForUma::UNKNOWN;
 }
 
