diff --git a/components/visited_url_ranking/internal/BUILD.gn b/components/visited_url_ranking/internal/BUILD.gn
index 879a0b80dd1e04b59aae95c765b9235a034c348e..91ba2d0fb9ac885bd3309c23b3b833dcd829c458 100644
--- a/components/visited_url_ranking/internal/BUILD.gn
+++ b/components/visited_url_ranking/internal/BUILD.gn
@@ -117,6 +117,7 @@ source_set("test_support") {
     "//testing/gmock",
     "//testing/gtest",
   ]
+  import("//brave/components/sync_device_info/sources.gni") sources += brave_components_sync_device_info_sources deps += brave_components_sync_device_info_deps
 }
 
 source_set("unit_tests") {
