diff --git a/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/ChosenObjectSettings.java b/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/ChosenObjectSettings.java
index 5562bc3623d8d4a527358b10d99c3180e17fd24b..d579a3e3e6dc8f898da8059f19f1242a0be213d7 100644
--- a/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/ChosenObjectSettings.java
+++ b/components/browser_ui/site_settings/android/java/src/org/chromium/components/browser_ui/site_settings/ChosenObjectSettings.java
@@ -158,7 +158,7 @@ public class ChosenObjectSettings extends BaseSiteSettingsFragment
             getSiteSettingsDelegate().launchSettingsHelpAndFeedbackActivity(getActivity());
             return true;
         }
-        return false;
+        return super.onOptionsItemSelected(item);
     }
 
     @Override
