diff --git a/components/search_engines/android/BUILD.gn b/components/search_engines/android/BUILD.gn
index 7fc78ce161432892253e5b6c72ddb8b8c6968488..f2776dbcba1d3fa48d209ae8a509bb3786921930 100644
--- a/components/search_engines/android/BUILD.gn
+++ b/components/search_engines/android/BUILD.gn
@@ -55,6 +55,7 @@ android_library("java") {
     "java/src/org/chromium/components/search_engines/TemplateUrl.java",
     "java/src/org/chromium/components/search_engines/TemplateUrlService.java",
   ]
+  deps += brave_components_search_engines_android_java_deps
 }
 
 android_library("features_java") {
