diff --git a/components/search_engines/BUILD.gn b/components/search_engines/BUILD.gn
index 3c6a3dc630b7253e796e8925f577c24f1c550d12..304b573f8fd889d51b3a026211f2bb69554e01d0 100644
--- a/components/search_engines/BUILD.gn
+++ b/components/search_engines/BUILD.gn
@@ -280,6 +280,7 @@ json_to_struct("prepopulated_engines") {
     ":search_engine_type",
     "//base",
   ]
+  additional_sources = brave_components_search_engines_sources
 }
 
 if (is_android) {
