diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index caad966c36298516c0b3899e264ccdf775aafb04..2a545c786f4072377fad4fa4485987a92fa6ce3d 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1262,6 +1262,7 @@ void PageInfo::PopulatePermissionInfo(PermissionInfo& permission_info,
 // applies to permissions listed in |kPermissionType|.
 bool PageInfo::ShouldShowPermission(
     const PageInfo::PermissionInfo& info) const {
+  BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
   // Note |ContentSettingsType::ADS| will show up regardless of its default
   // value when it has been activated on the current origin.
   if (info.type == ContentSettingsType::ADS) {
