diff --git a/components/sync/service/sync_service_impl.h b/components/sync/service/sync_service_impl.h
index 59fb52c8800ff57aa9965d96f54dfb33a4db179d..c339a94512eda62c63406c456d61976c420167de 100644
--- a/components/sync/service/sync_service_impl.h
+++ b/components/sync/service/sync_service_impl.h
@@ -256,6 +256,7 @@ class SyncServiceImpl : public SyncService,
   void ReportDataTypeErrorForTest(DataType type);
 
  private:
+  BRAVE_SYNC_SERVICE_IMPL_H_
   enum UnrecoverableErrorReason {
     ERROR_REASON_ENGINE_INIT_FAILURE,
     ERROR_REASON_ACTIONABLE_ERROR,
