diff --git a/components/sync/service/BUILD.gn b/components/sync/service/BUILD.gn
index e6e0b0f74c0147d7ca7732fd9aad936b033d20c8..1d7ddc3f9b15641a2b14b5d451cebe7b315790f7 100644
--- a/components/sync/service/BUILD.gn
+++ b/components/sync/service/BUILD.gn
@@ -130,4 +130,5 @@ static_library("service") {
   }
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
+  sources += brave_components_sync_driver_sources deps += brave_components_sync_driver_deps
 }
