diff --git a/components/sync/service/sync_service_impl.h b/components/sync/service/sync_service_impl.h
index b9802af2623b2c4f7e37e6c3f298560d0c2564fe..73978686b2d2139f8f199c9ff204a72833f773d5 100644
--- a/components/sync/service/sync_service_impl.h
+++ b/components/sync/service/sync_service_impl.h
@@ -273,6 +273,7 @@ class SyncServiceImpl : public SyncService,
   size_t GetQueuedLocalDataMigrationItemCountForTest() const;
 
  private:
+  BRAVE_SYNC_SERVICE_IMPL_H_
   enum UnrecoverableErrorReason {
     ERROR_REASON_ENGINE_INIT_FAILURE,
     ERROR_REASON_ACTIONABLE_ERROR,
