diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index 2f964d28cbca483775acff6332e2cf90f824bb91..9d9f4a41ecc096467225f265a4cec58b3a02a2bc 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1243,6 +1243,7 @@ void PageInfo::PopulatePermissionInfo(PermissionInfo& permission_info,
 // applies to permissions listed in |kPermissionType|.
 bool PageInfo::ShouldShowPermission(
     const PageInfo::PermissionInfo& info) const {
+  BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
   // Note |ContentSettingsType::ADS| will show up regardless of its default
   // value when it has been activated on the current origin.
   if (info.type == ContentSettingsType::ADS) {
