diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index ee169c534554112e82b3db46fc682187e901b52a..a3f80f309123d0cc3e7d4a918d86e7c5ac6c8e0a 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3548,6 +3548,7 @@ source_set("browser") {
       "profiling_utils.cc",
     ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
