diff --git a/content/public/browser/BUILD.gn b/content/public/browser/BUILD.gn
index 302c581cf3da182d2f39e9b7038077ebae6944db..bc152132ee674ed38cd43137d8b7a813892ded93 100644
--- a/content/public/browser/BUILD.gn
+++ b/content/public/browser/BUILD.gn
@@ -741,6 +741,7 @@ source_set("browser_sources") {
   if (is_chromeos_lacros) {
     deps += [ "//chromeos/startup" ]
   }
+  import("//brave/content/public/browser/sources.gni") sources += brave_content_public_browser_sources
 }
 
 if (is_android) {
