diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 4502ba5b27ebd7d5d97ba10d8da4844eb9a0f777..bd5037e5a13473ed0913a4d253138e2359b048c2 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3534,6 +3534,7 @@ source_set("browser") {
       "profiling_utils.cc",
     ]
   }
+  import("//brave/content/browser/sources.gni") deps += brave_content_browser_deps
 }
 
 if (is_android) {
