diff --git a/content/public/browser/BUILD.gn b/content/public/browser/BUILD.gn
index 1c1d59004b1e0f0cd8f14f7f609f42319b92d536..4ad87aadbe181aec43b97c09c2988ee5d44fbccb 100644
--- a/content/public/browser/BUILD.gn
+++ b/content/public/browser/BUILD.gn
@@ -744,6 +744,7 @@ source_set("browser_sources") {
   if (enable_video_effects) {
     public_deps += [ "//services/video_effects/public/mojom:mojom" ]
   }
+  import("//brave/content/public/browser/sources.gni") sources += brave_content_public_browser_sources
 }
 
 if (is_android) {
