diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 70e143feb497711d12857333d48ce2ac4325bcd9..a29606c38a849359bb2b81c387f1938615f9cc41 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3533,6 +3533,7 @@ source_set("browser") {
       "profiling_utils.cc",
     ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
