diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 431d8486fcf5f6895ff238b0d37e897dea85aed5..b6ff2736d3a4fefd21eec4b02b0022c46cc12a86 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3686,6 +3686,7 @@ source_set("browser") {
   if (is_chromeos) {
     deps += [ "//ash/constants:constants" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
