diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index 978b771ce4b4bf133d5f8e8bcab7615598110727..0870bfdb66d51fe4ddfc90ea18c247346cc3434a 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -935,6 +935,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
