diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index 5ac9504206e05d5b81a26df160eff98d0f6cbbb7..86dc8f560772a3694a8bf1febcdca21b6f424271 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -921,6 +921,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
