diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 16a57714f7f41ccdf3d3ca10d9d59fe5fa97d19a..2ab259184af8755bad5d4199f94d55f674875dea 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3567,6 +3567,7 @@ source_set("browser") {
       "profiling_utils.cc",
     ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
