diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 75e78648f7e0e5290beb6fec331b8d09f5c540fb..0c8f9ead628d03b5baec5b01c0f103ffcd100cdc 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3538,6 +3538,7 @@ source_set("browser") {
       "profiling_utils.cc",
     ]
   }
+  import("//brave/content/browser/sources.gni") deps += brave_content_browser_deps
 }
 
 if (is_android) {
