diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index 09cc182f71290dc622d17fcbd1d6e533a3eabb75..66665620dac1fc2d1d25f27d6f79468f8561a414 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -904,6 +904,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
