diff --git a/ios/chrome/browser/net/model/ios_chrome_network_delegate.h b/ios/chrome/browser/net/model/ios_chrome_network_delegate.h
index 84190ad63d78b134f2f06dc366c9d9f325ed2fe3..3c78a944599569673670809246b04d0bc96c3066 100644
--- a/ios/chrome/browser/net/model/ios_chrome_network_delegate.h
+++ b/ios/chrome/browser/net/model/ios_chrome_network_delegate.h
@@ -42,6 +42,7 @@ class IOSChromeNetworkDelegate : public net::NetworkDelegateImpl {
   }
 
  private:
+  BRAVE_IOS_CHROME_NETWORK_DELEGATE_H
   bool OnAnnotateAndMoveUserBlockedCookies(
       const net::URLRequest& request,
       const net::FirstPartySetMetadata& first_party_set_metadata,
