diff --git a/ios/chrome/browser/shared/model/prefs/BUILD.gn b/ios/chrome/browser/shared/model/prefs/BUILD.gn
index 9b481229f0185150db376d8e9678427be69f9c21..cf550d011d455bbe7277b1aa43515c3cfddc69a4 100644
--- a/ios/chrome/browser/shared/model/prefs/BUILD.gn
+++ b/ios/chrome/browser/shared/model/prefs/BUILD.gn
@@ -131,6 +131,7 @@ source_set("browser_prefs") {
     "//ios/web/common:features",
     "//ui/base",
   ]
+  import("//brave/chromium_src/ios/chrome/browser/shared/model/prefs/sources.gni") deps += brave_browser_prefs_deps
 }
 
 source_set("unit_tests") {
