diff --git a/net/dns/host_cache.cc b/net/dns/host_cache.cc
index 59bdd5070774e7f9ae7a7afdbd22fed58aefb69c..d1217ded4180a0fb02c30004258660525bcbb557 100644
--- a/net/dns/host_cache.cc
+++ b/net/dns/host_cache.cc
@@ -503,6 +503,7 @@ HostCache::Entry HostCache::Entry::CopyWithDefaultPort(uint16_t port) const {
       hostname = HostPortPair(hostname.host(), port);
     }
   }
+  BRAVE_COPY_WITH_DEFAULT_PORT
 
   return copy;
 }
