diff --git a/third_party/tflite/BUILD.gn b/third_party/tflite/BUILD.gn
index cb632ec7d5bb32ff499aaa76759578f2a1ea8825..7e2909a20f0aa095ae6c697d16f197130623d2db 100644
--- a/third_party/tflite/BUILD.gn
+++ b/third_party/tflite/BUILD.gn
@@ -237,6 +237,7 @@ tflite_static_library("tflite_builtin_op_resolver") {
     "//services/webnn/*",
     "//third_party/mediapipe/*",
   ]
+  import("//brave/third_party/tflite/tflite.gni") visibility += brave_tflite_builtin_op_resolver_visibility
 
   conditional_deps = [ ":tflite" ]
 }
