diff --git a/third_party/rust/rand/v0_8/BUILD.gn b/third_party/rust/rand/v0_8/BUILD.gn
index e494288a370cb33f715319a4a36d7224527775ae..2e5682582d08a1ea24d9cfbf19ce50f905a816bc 100644
--- a/third_party/rust/rand/v0_8/BUILD.gn
+++ b/third_party/rust/rand/v0_8/BUILD.gn
@@ -78,5 +78,5 @@ cargo_crate("lib") {
     "--cap-lints=allow",  # Suppress all warnings in crates.io crates
   ]
 
-  testonly = true
+  features += [ "small_rng" ]
 }
