diff --git a/third_party/tflite/BUILD.gn b/third_party/tflite/BUILD.gn
index 78a63109a0d926cd45fa692571c23557999623e0..1c341275852ec8279291bd0c9c1288cb302f6112 100644
--- a/third_party/tflite/BUILD.gn
+++ b/third_party/tflite/BUILD.gn
@@ -211,6 +211,7 @@ tflite_static_library("tflite_builtin_op_resolver") {
     "//services/webnn/*",
     "//third_party/mediapipe/*",
   ]
+  import("//brave/third_party/tflite/tflite.gni") visibility += brave_tflite_builtin_op_resolver_visibility
 
   conditional_deps = [ ":tflite" ]
 }
