diff --git a/third_party/blink/renderer/core/loader/mixed_content_checker.cc b/third_party/blink/renderer/core/loader/mixed_content_checker.cc
index 467ab0bd717d8b8755a2979ccfea2866cbc296bf..642a996f33bf955814980f508a7c924913499e9a 100644
--- a/third_party/blink/renderer/core/loader/mixed_content_checker.cc
+++ b/third_party/blink/renderer/core/loader/mixed_content_checker.cc
@@ -274,6 +274,7 @@ bool RequestIsSubframeSubresource(Frame* frame) {
 // static
 bool MixedContentChecker::IsMixedContent(const SecurityOrigin* security_origin,
                                          const KURL& url) {
+  BRAVE_MIXED_CONTENT_CHECKER_IS_MIXED_CONTENT
   return IsMixedContent(
       security_origin->GetOriginOrPrecursorOriginIfOpaque()->Protocol(), url);
 }
