diff --git a/tools/web_dev_style/eslint.py b/tools/web_dev_style/eslint.py
index a20ec36544236605371855e5caefe3f4229ddf5c..89472cb4baeec3f8236f96d2c9ab44e51f74d83d 100755
--- a/tools/web_dev_style/eslint.py
+++ b/tools/web_dev_style/eslint.py
@@ -28,6 +28,7 @@ def Run(os_path=None, args=None):
   ] + args)
 
 
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
 if __name__ == '__main__':
   import os
   import sys
