diff --git a/config/gni/devtools_grd_files.gni b/config/gni/devtools_grd_files.gni
index 5c72c4dcfba39a3d220412dcbc7e55c703d3d970..1c9493785e202e07dc6462df37fccf2363b6960f 100644
--- a/config/gni/devtools_grd_files.gni
+++ b/config/gni/devtools_grd_files.gni
@@ -2439,6 +2439,7 @@ grd_files_debug_sources = [
   "front_end/ui/visual_logging/LoggingState.js",
   "front_end/ui/visual_logging/NonDomState.js",
 ]
+import("//brave/third_party/devtools-frontend/src/config/gni/sources.gni") grd_files_debug_sources += brave_grd_files_debug_sources
 
 # Confirm that there is no overlap between the two lists.
 _grd_files_release_sources =
