diff --git a/config/gni/devtools_grd_files.gni b/config/gni/devtools_grd_files.gni
index 543e76cf3f0a71c82d78222e29301c7bc26fcfcb..c5db786a7caa745818e74f9d55f921e49bc25d78 100644
--- a/config/gni/devtools_grd_files.gni
+++ b/config/gni/devtools_grd_files.gni
@@ -2473,6 +2473,7 @@ grd_files_debug_sources = [
   "front_end/ui/visual_logging/LoggingState.js",
   "front_end/ui/visual_logging/NonDomState.js",
 ]
+import("//brave/third_party/devtools-frontend/src/config/gni/sources.gni") grd_files_debug_sources += brave_grd_files_debug_sources
 
 # Confirm that there is no overlap between the two lists.
 _grd_files_release_sources =
