diff --git a/BUILD.gn b/BUILD.gn
index 1e07990b260c755b014e69aecbd06068356dd707..9ad7a800185cbd2892aa51ac46cf276cf6524927 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -784,6 +784,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3270,6 +3271,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
