diff --git a/BUILD.gn b/BUILD.gn
index 93ac330897997e6fd0f36cabc2a13b45367a2fd9..f155c9321f92731bfc73cb2e5dea39abcfe96137 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -850,6 +850,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3240,6 +3241,7 @@ v8_header_set("v8_headers") {
     ":v8_tracing",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
