diff --git a/BUILD.gn b/BUILD.gn
index cc9e9b7a190d38413f850ace6cb615f807492eb2..7227caa960db16341376c9b66de07b062f5e0503 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -785,6 +785,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3262,6 +3263,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
