diff --git a/BUILD.gn b/BUILD.gn
index 6893332332a11b79d34080573d33ce4a5134bf5d..c8995fd405818cf09c6216211254226e1cfdd1e9 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -850,6 +850,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3247,6 +3248,7 @@ v8_header_set("v8_headers") {
     ":v8_tracing",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
