diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 3a148fa73f4cb19f07240b402e200c3006058c97..8b2766886e648fd06bd74ccbdee87c935b3eaea0 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3385,6 +3385,7 @@ MaybeDirectHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   CHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
