diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 2e285596b2b61ad31291901ebd725e52633cc839..32a2c154f15ad2ae1fc8402b45ec7e573fb7e008 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3394,6 +3394,7 @@ MaybeDirectHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   CHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
