diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 6bde6de2e525b93727e977ed1639aeba60e757ac..3a9e852d9a05c951db26312994a5c3f00cedbaa0 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3292,6 +3292,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
