diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index d86a3e8b7937d9fe341f723bbe67f46eddc7e8c8..494c9bb7a7b6fbeb06f716057583c94a37d9fc04 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3340,6 +3340,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   CHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
