diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index b8bb9fe270eb11228814e6d6984cb41ae39514ed..6b1bcb7c984bb11b9ac4c67eaafd1fdec7d7ca06 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3279,6 +3279,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
