diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 71ffe1e6f2ae60046cca01d55cb5a8a1c2689bc3..1a51c782781a8ffc3689f83ae787d9366b19ae88 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3314,6 +3314,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
